test/unit/resolve-hook/input.js

```diff
@@ -0,0 +1,2 @@
+require('./dep');
+require('when');
```

Comment:
I don't think this test should depend on `when`, its not necessary to test the hook.

Reply:
https://github.com/vercel/nft/pull/153#discussion_r485210888